Run wmluiltok, wml and wmldbcreate from PATH: the target build cannot run the cross-compiled ones, so the host-built tools are used.

--- a/tools/wml/Makefile.am
+++ b/tools/wml/Makefile.am
@@ -51,15 +51,15 @@ $(WMLTARGETS): wml-uil.mm
 
 wml-uil.mm: wmluiltok wml UilLexPars.c $(TABLE)
 	$(RM) $(WMLTARGETS) wml.report wml-uil.mm
-	./wmluiltok <$(srcdir)/Uil.y >tokens.dat
+	wmluiltok <$(srcdir)/Uil.y >tokens.dat
 	$(RM) wml-uil.mm
-	./wml $(srcdir)/$(TABLE)
+	wml $(srcdir)/$(TABLE)
 	$(RM) tokens.dat
 
 all: motif.wmd
 
 motif.wmd: wmldbcreate
-	./wmldbcreate -o motif.wmd
+	wmldbcreate -o motif.wmd
 
 clean-local:
 	$(RM) $(WMLTARGETS) $(WMDTABLE) $(REPORT) lex.yy.c libwml.la wml \
